include/s3select.h

```diff
@@ -2199,6 +2250,7 @@ class base_s3object
             }
 
           result_values_to_string(projections_resuls,result);
+          result.append(output_row_delimiter);
```

Comment:
`result.append(output_row_delimiter);` is something that should be done 
in `result_values_to_string`
the `getMatchRow` should only compute projections